src/locales/it.json

```diff
@@ -21,7 +21,7 @@
 
   "stepFeaturesTitle": "Selezione funzionalità",
   "stepFeaturesBoardDesc": "Seleziona il tipo di scheda su cui è basato il progetto",
-  "stepFeaturesBoardGeneric": "Generic",
+  "stepFeaturesBoardGeneric": "Generica",
```

Comment:
This one is not used and will probably be removed/replaced. Please read the discuassion on PR #185.

Reply:
@benzino77 

Currently "Generic" is used in the GUI.

I propose you to merge current PR and when you will remove in master language I will update the Italian language 

Thanks.